### ISSUE TITLE
How I can get Panoramic HDR (like rectangle, not dual fish eye)?

### ISSUE THREAD

**Issue by mobile-sergey** [Insta360Develop/CameraSDK-Android#79]
We need Panoramic HDR for processing it on our library in Android app. Our lib can't work with dual fish eye.
But in documentation and in demo-base I can't found how I can do it.

I try get photo by methods:
`InstaCameraManager.getInstance().startHDRPanoCapture(FOCUS_SENSOR_ALL, false)`

`InstaCameraManager.getInstance().startHDRCapture(false)`

`InstaCameraManager.getInstance().肌緭().startHDRCapture(intArrayOf(0, -1, 1), InstaCameraManager.FUNCTION_MODE_HDR_CAPTURE, byteArrayOf(), 0)`

After any of this methods I get 2-3 `.insp` photos and raw files (if I use `true` in params)

I try to use function generateHDR like this:

```
        val workWrapper = WorkWrapper(urls.toTypedArray())

        val hdrFile = File(
            Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES).toString()
                    + "/" + File.separator + urls.first().substringAfterLast("/") + ".hdr.jpg"
        )
        withContext(Dispatchers.IO) {
            hdrFile.createNewFile()
        }
        val hdrOutputPath = hdrFile.absolutePath
        val isSuccessful = StitchUtils.generateHDR(workWrapper, hdrOutputPath)
        if (isSuccessful) {
                    return urls.first()
        }
```
Where `urls` - is paths to HDR files - is Set, that i fill on override `onCaptureFinish(filePaths: Array<String>)` in `ICaptureStatusLisnter`

But after any of first methods and this algorithm I get dual fish eye HDR - not rect panoramic HDR.

I try to use export like this:

```
            val exportPath = hdrFile.path
            val builder = ExportImageParamsBuilder()
                .setExportMode(ExportUtils.ExportMode.PANORAMA)
                .setTargetPath(exportPath)
                .setUrlForExport(hdrOutputPath)
            return builder.urlForExport
```

But again I get dual fish eye HDR - not rect panoramic HDR.

I try to use `StitchUtils.stitchSeparatedFisheyeFile(WorkWrapper workWrapper, String outputPath)` by adding to workWrapper 2 HDR images from `InstaCameraManager.getInstance().startHDRPanoCapture(FOCUS_SENSOR_FRONT, false)` and `InstaCameraManager.getInstance().startHDRPanoCapture(FOCUS_SENSOR_REAR, false)` - coping this files to outputPath on phone, but nothing happening.

If I use simple images instead of HDR and run StitchUtils.stitchSeparatedFisheyeFile() - I get app crush with SEGFAULT in you libs.

In your documentation and demobase I found only one place when I can change dual fish eye to panoramic rect - in Preview. But it in interface (but I need to processing it on data layer) and I think its not HDR

How I can get rect panoramic HDR on data layer with your SDK?

**Comment by ad1tya2** [comment]
Were you able to find a solution for this?

**Comment by mobile-sergey** [comment]
No, I don't find solution. In our project we cancel using this library

**Comment by Double-Insta360** [comment]
Sorry for getting back to you late. We recommend referring to the latest release of our Android Media SDK for further details.
